<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntegrationWindow</class>
 <widget class="QMainWindow" name="IntegrationWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>600</width><height>400</height></rect>
  </property>
  <property name="windowTitle">
   <string>시스템 연동</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="syncLabel">
      <property name="text">
       <string>연동 상태: 대기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adminSyncBtn">
      <property name="text">
       <string>Admin 동기화</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mobileWebLabel">
      <property name="text">
       <string>모바일/웹 연동: 미연동</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connectMobileWebBtn">
      <property name="text">
       <string>모바일/웹 연결</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="rfidLabel">
      <property name="text">
       <string>RFID 연계: 비활성화</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rfidLinkBtn">
      <property name="text">
       <string>RFID 연계 활성화</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>